testsuite/cluster-test/src/main.rs

```diff
@@ -222,10 +234,13 @@ struct ClusterTestRunner {
     health_check_runner: HealthCheckRunner,
     deployment_manager: DeploymentManager,
     experiment_interval: Duration,
-    slack: Option<SlackClient>,
+    slack: SlackClient,
+    slack_log_url: Option<Url>,
```

Comment:
Can we move these URLs into SlackClient. This struct is getting very big.

slack_message and slack_changelog_message can also be made methods of SlackClient